Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/messaging/service/decorator/cxf/TLSUddiClientEndpointDecorator.java

```diff
@@ -0,0 +1,65 @@
+/**
+ *
+ */
+package gov.hhs.fha.nhinc.messaging.service.decorator.cxf;
+
+import org.apache.cxf.configuration.jsse.TLSClientParameters;
+import org.apache.cxf.transport.http.HTTPConduit;
+
+import gov.hhs.fha.nhinc.properties.PropertyAccessor;
+import gov.hhs.fha.nhinc.messaging.service.ServiceEndpoint;
+import gov.hhs.fha.nhinc.nhinclib.NhincConstants;
+import gov.hhs.fha.nhinc.properties.PropertyAccessException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.util.StringUtils;
+
+/**
+ * @author mpnguyen
+ *
+ */
+public class TLSUddiClientEndpointDecorator<T> extends TLSClientServiceEndpointDecorator<T> {
```

Comment:
Within the CONNECT codebase, pick and choose whether acronyms will be camel-cased or not -- I strongly suggest not mixing and matching, e.g.:

Mixed (bad):  TLSUddi
Not mixed (okay):  TlsUddi
Not mixed (okay):  TLSUDDI